Jenkinsfile

```
@@ -25,6 +25,9 @@ app.build(
                 memory: '10Gi',
             ]
         ],
+        storageConfig: [
+          size: '10Gi',
```

```suggestion
          size: '15Gi',
```

default is already 10Gi. Before we merge this we'll want to clean up the old caches

---

The docs say default is 5Gi.

How can I find utilisation metrics, for CPU and memory also? Inline is easy, remote builder is hidden and affecting other projects like NitroID.

---

Actually, default seems to be 10Mi. Typo?

https://github.com/powerhome/pac-buildkit/blob/main/charts/operator/templates/configmap.yaml#L16-L18

---

The 10Mi ephemeral storage limit so far has only come into play on the multi-arch builds. This is using [this default](https://github.com/powerhome/pac-buildkit/blob/main/charts/operator/templates/configmap.yaml#L9) in question

---

```
#39 [release 4/6] RUN --mount=id=playbook-yarncache,type=cache,target=/home/app/.cache/yarn,uid=9999,gid=9999,sharing=locked     cd playbook; NODE_ENV=production; yarn release

#39 ERROR: open /var/lib/buildkit/runc-overlayfs/snapshots/snapshots/124/fs/usr/local/share/.cache/yarn/v6/npm-prosemirror-model-1.19.0-d7ad9a65ada0bb12196f64fe0dd4fc392c841c29-integrity/node_modules/prosemirror-model/src/index.ts: no space left on device
```

Is this ephemeral storage exhaustion, or the cache volume?

---

As for utilization metrics, it's on the todo list to update the links to also include remote metrics, but you can hunt it down via the runner pod name: https://metrics.powerapp.cloud/d/s9c8D_K7k/pod-resource-usage?orgId=1&refresh=10s&var-datasource=000000003&var-namespace=pac-buildkit&var-filter=.%2B&var-pod=buildkit-runner-mssns-tydiw&from=now-1h&to=now